### PR TITLE
Link Up gradation

### PR DESCRIPTION
**Pull request by joinimran**
Missing links have been updated.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6174

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md [hunk 1]
```diff
@@ -118,8 +118,8 @@ After the task sequence finishes, the computer will be fully upgraded to Windows
 
 With Configuration Manager, new built-in functionality makes it easier to upgrade to Windows 10.
 
-**Note**  
-For more details about Configuration Manager, see the [Configuration Manager Team blog](https://go.microsoft.com/fwlink/p/?LinkId=620205). An [evaluation version is currently available](https://go.microsoft.com/fwlink/p/?LinkId=620206) for you to try. The instructions below are specific to the Technical Preview 2 release and may change after the next version of Configuration Manager is released.
+> [!Note]
```
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md]:
```suggestion
> [!NOTE]
```

File: windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md [hunk 2]
```diff
@@ -118,8 +118,8 @@ After the task sequence finishes, the computer will be fully upgraded to Windows
...
+> For more details about Configuration Manager, see the [Configuration Manager Team blog](https://techcommunity.microsoft.com/t5/configuration-manager-blog/bg-p/ConfigurationManagerBlog). An [evaluation version is currently available](https://www.microsoft.com/en-us/evalcenter/evaluate-system-center-configuration-manager-and-endpoint-protection-technical-preview) for you to try. The instructions below are specific to the Technical Preview 2 release and may change after the next version of Configuration Manager is released.
```
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md]:
```suggestion
> For more details about Configuration Manager, see the [Configuration Manager blog](https://techcommunity.microsoft.com/t5/configuration-manager-blog/bg-p/ConfigurationManagerBlog). An [evaluation version](https://www.microsoft.com/en-us/evalcenter/evaluate-system-center-configuration-manager-and-endpoint-protection-technical-preview) is currently available for you to try. The instructions below are specific to the Technical Preview 2 release and may change after the next version of Configuration Manager is released.
```

File: windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md [hunk 3]
```diff
@@ -207,7 +207,7 @@ After the task sequence completes, the computer will be fully upgraded to Window
 
 [Windows 10 deployment scenarios](../windows-10-deployment-scenarios.md)
 
-[Configuration Manager Team blog](https://go.microsoft.com/fwlink/p/?LinkId=620109)
+[Configuration Manager Team blog](https://techcommunity.microsoft.com/t5/configuration-manager-blog/bg-p/ConfigurationManagerBlog)
```
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md]:
```suggestion
[Configuration Manager blog](https://techcommunity.microsoft.com/t5/configuration-manager-blog/bg-p/ConfigurationManagerBlog)
```

File: windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/upgrade-to-windows-10-with-system-center-configuraton-manager.md]:
```suggestion
> For more details about Configuration Manager, see the [Configuration Manager blog](https://techcommunity.microsoft.com/t5/configuration-manager-blog/bg-p/ConfigurationManagerBlog). An [evaluation version](https://www.microsoft.com/en-us/evalcenter/evaluate-system-center-configuration-manager-and-endpoint-protection-technical-preview) is currently available for you to try. The instructions below are specific to the Technical Preview 2 release and may change after the next version of Configuration Manager is released.
```

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by greg-lindsay** [comment]
Sorry but this entire doc set has been rewritten. I'm pushing out a new PR today, so all these changes are moot.

**Pull request closed without merging**